### PR TITLE
Add collaborators

### PR DESCRIPTION
**Pull request by fmvilas**

### CODE REVIEW

File: site/index.html [hunk 1]
```diff
@@ -534,6 +534,35 @@ <h5>Bio</h5>
...
+                  <h4>Andrzej Jarzyna</h4>
+                  <h6>Adidas</h6>
+                </div>
+                <div class="col-md-3"><img src="/images/collaborators/jonas-lagoni.webp" style="width:80%;border-radius:100%;object-fit:cover;margin-bottom:20px;">
+                  <h4>Jonas Lagoni</h4>
+                  <h6>EURISCO</h6>
+                </div>
+                <div class="col-md-3"><img src="/images/collaborators/mike-ralphson.png" style="width:80%;border-radius:100%;object-fit:cover;margin-bottom:20px;">
+                  <h4>Mike Ralphson</h4>
+                  <h6>Stoplight</h6>
```
Comment (MikeRalphson) [inline site/index.html:555]: Can this be changed to "Mermade Software"? I'm no longer at Stoplight.

File: sources/pug/pages/index.pug [hunk 2]
```diff
@@ -214,4 +214,30 @@ block page
...
+              h4 Andrzej Jarzyna
+              h6 Adidas
+            .col-md-3
+              img(src='/images/collaborators/jonas-lagoni.webp', style='width:80%;border-radius:100%;object-fit:cover;margin-bottom:20px;')
+              h4 Jonas Lagoni
+              h6 EURISCO
+            .col-md-3
+              img(src='/images/collaborators/mike-ralphson.png', style='width:80%;border-radius:100%;object-fit:cover;margin-bottom:20px;')
+              h4 Mike Ralphson
+              h6 Stoplight
```
Comment (MikeRalphson) [inline sources/pug/pages/index.pug:237]: Ditto.

File: site/index.html [hunk 3]
```diff
@@ -534,6 +534,31 @@ <h5>Bio</h5>
           </div>
         </div>
       </section>
+      <!-- Section Collaborators-->
+      <section class="section section-lg bg-default text-center">
```
Comment (derberg) [inline site/index.html:538]:
```suggestion
      <section id="collaborators" class="section section-lg bg-default text-center">
```

Comment (fmvilas) [reply site/index.html]: Done!

**Pull request merged**